packages/botonic-plugin-contentful/src/tools/l10n/csv-export.ts

```diff
@@ -68,8 +72,7 @@ export class CsvExport {
   }
 
   async *generate(cms: CMS, from: Locale): AsyncGenerator<CsvLine> {
-    console.error('TODO: pending to export URL models')
-    for (const model of BOTONIC_CONTENT_TYPES) {
+    for (const model of [...BOTONIC_CONTENT_TYPES, ContentType.URL]) {
```

Comment:
TODO: add in "MessageContentType" the URL type, so it can be used in other parts in the future?

Reply:
would be nice, but since URL is not a React message, I decided not to do it